api/src/templates/new_member.body.html

```diff
@@ -1,167 +1,194 @@
+        <li>&#128213; <a href="https://wiki.makerspace.se/V%C3%A4lkommen_till_Stockholm_Makerspace">Wikisidor</a></li>
+    </ul>
+    <li>Kolla gärna vad andra medlemmar har delat på <a href="https://stockholmmakerspace.slack.com/archives/C4M0MJ96K">#projekt</a>-kanalen på Slack och få inspiration!</li>
+    <li>Gå med i <a href="https://stockholmmakerspace.slack.com/archives/CGQ04052Q">#events</a>-kanalen på Slack för att få reda på kommande kurser, sammankomster och andra evenemang.</li>
+</ul>
 
-<h3>Jag bygger något större än labblådan, var ska jag förvara det?</h3>
-<p>I mån av plats så kan du lägga ditt projekt i de avsedda hyllorna i lagerrummet, under max 3 månader. Ditt projekt ska vara märkt med en etikett från <a href="https://wiki.makerspace.se/Memberbooth">Memberbooth</a>, annars kommer det att slängas. Vi har plats för köpta skivor av plywood, akryl och liknande bredvid laserskäraren i stora rummet. Alla regler kring förvaring på Stockholm Makerspace går att hitta på <a href="https://wiki.makerspace.se/Medlemsf%C3%B6rvaring">wikin.</a></p>
+<h3>Trivselregler</h3>
 
-<h3>Jag lämnade mitt projekt på ett bord över natten och sen var det borta!</h3>
-<p>Du måste städa efter dig. Vi har en städare som vi betalar en ansenlig summa för att våttorka golv och bord varje vecka. Men han rör inte bord med saker på, och städar inte golv där det står saker. Det är jätteviktigt att du städar undan efter dig, så att vi kan ha en gemensam, ren lokal. Har du råkat lämna saker på ett bord så kan det ha slängts eller flyttats till en Lost&Found-låda. Medlemmar hjälps åt att tömma soporna. Fråga en medlem om du inte vet hur man gör, eller kolla in <a href="https://wiki.makerspace.se/St%C3%A4drutiner">städrutinerna</a>.</p>
+<p style="text-align:center;"><i>“Visa hänsyn. Ta ansvar.”</i></p>
 
-<h3>Jag är hungrig.</h3>
-<p>Stockholm Makerspace erbjuder ett pentry med en kaffekokare och mikrovågsugn. Du kan också köpa färdigmat på The Coffice över gatan, eller på 7-Eleven eller T-Snabben nere vid Östra Station.</p>
+<p>Visa hänsyn mot andra medlemmar:</p>
+<ul>
+    <li>Städa alltid upp efter dig. Glöm inte att fler medlemmar kommer senare för att använda samma ytor och maskiner. Den medlemmen kan vara du.</li>
+    <li>Se till att använda utrustning och lokaler på ett korrekt sätt. Om utrustning används på felaktivt vis kan du skada utrustningen och göra den omöjlig att använda för fler medlemmar. Den medlemmen kan vara du.</li>
+    <li>Informera om något är trasigt, försvunnet eller om det behöver fyllas på med något. Saker går sönder ibland, det är sånt som händer. Om du inte kan laga det själv så skriv på Slack. Om du inte gör det kommer utrustningen förbli trasig när fler medlemmar vill använda den. Den medlemmen kan vara du.</li>
+    <li>Ta inte utrustning med dig. Om du tar hem saker är den inte tillgänglig för fler medlemmar. Den medlemmen kan vara du.</li>
```

Comment:
Att inte ta med utrustning från spacet är inte bara att visa hänsyn, det är strikt förbjudet enligt makerspace regler.

Reply:
Good point. 👍
We thought it was good to mention it. Would you put it in some other part of the email?